go/libkb/logout.go

```diff
@@ -190,9 +190,9 @@ func CanLogout(mctx MetaContext) (res keybase1.CanLogoutRes) {
 		}
 	}
 
-	hasRandomPW, err := LoadHasRandomPw(mctx, keybase1.LoadHasRandomPwArg{
-		ForceRepoll: false,
-	})
+	prefetcher := mctx.G().GetHasRandomPWPrefetcher()
+	forceRepoll := prefetcher == nil || !prefetcher.prefetched
+	passphraseState, err := LoadPassphraseStateWithForceRepoll(mctx, forceRepoll)
```

Comment:
can we have a comment here that this is in case prefetcher did not finish yet or for standalone mode?